### ISSUE TITLE
libInjection reports incorrect numbers

### ISSUE THREAD

**Issue by RedXanadu** [coreruleset/go-ftw#392]
For quantitative testing mode, the numbers reported for libInjection rules do not match with the numbers produced by running the reference shell script against Apache + ModSecurity v2.

<INSERT EXAMPLES/DATA HERE>

**Comment by RedXanadu** [comment]
It has been suggested that Coraza is compiling a *different version of libInjection* and may even be working using *different data sets* to mainstream ModSecurity. This may explain why the Coraza version is hiding lots of false positives. Therefore, the quantitative testing numbers will be misleading and will mask (user) problems with libInjection-based rules.

**Comment by fzipi** [comment]
@RedXanadu can you rerun these tests? There have been lots of fixes in the libinjection version from coraza, many addressing this problem.

**Comment by fzipi** [comment]
Traced this down while looking at [libinjection-go#118](https://github.com/corazawaf/libinjection-go/issues/118) (a related report from the same corpus). The libinjection-version/dataset explanation isn't it — this is a `SecUnicodeMapFile` config difference between the reference platform and Coraza, and it's empirically confirmed.

## Root cause

The reference container `quantitative-testing.sh` uses (`owasp/modsecurity-crs:apache`, from `modsecurity-crs-docker`) bakes in `MODSEC_UNICODE_MAPPING=20127` (`apache/Dockerfile:117`), which sets:

```
SecUnicodeMapFile unicode.mapping 20127
```

unconditionally in `modsecurity.conf`. Codepage `20127` (US-ASCII) in `unicode.mapping` contains:

```
2018:27 2019:27 201a:2c 201c:22 201d:22 201e:22
```

i.e. curly quotes/apostrophes (`'`/`'`/`"`/`"`) are mapped straight to ASCII `'`/`'`/`"` by `t:urlDecodeUni`'s best-fit table **before libinjection ever sees the string**. Coraza has no equivalent best-fit table, so it never performs this normalization — it's not a gap in Coraza's libinjection port, it's a transform the reference platform applies that Coraza doesn't.

## Empirical confirmation

Ran the exact reference image locally, once as-is and once with `SecUnicodeMapFile` removed from `modsecurity.conf` (same image, `docker exec`-verified clean startup, PL1 default, rule 942100 active in both):

```
payload 1187 ("Both business and consumer confidence...", raw UTF-8 “ U+201C)
  mapping ON  -> HTTP 403, libinjection fingerprint 's&nof', Matched Data shows \x22 (i.e. literal ")
  mapping OFF -> HTTP 503 (WAF did not block; request passed to the — absent — backend)

payload from libinjection-go#118 ("And he is now the governor of Georgia.", same raw curly quote)
  mapping ON  -> HTTP 403, fingerprint 's&nof'
  mapping OFF -> HTTP 503 (not flagged)

control: literal ASCII " prefix (no curly quote involved)
  mapping ON  -> HTTP 403
  mapping OFF -> HTTP 403  (unaffected, as expected — nothing to normalize)
```

With the unicode map disabled, the reference platform's behavior on the curly-quote payloads matches what `ftw + Coraza` already reports (no match) — while payloads that don't depend on curly-quote normalization (e.g. #5674, no quote characters at all) are flagged identically either way, consistent with the "SUCCESSFUL MATCH" case already reported in this issue.

## Suggestion

For apples-to-apples quantitative comparisons against Coraza, the reference container should run without `SecUnicodeMapFile` (or Coraza would need its own best-fit Unicode table to match parity — a separate feature question, not a bug fix). I'd suggest re-running the 10K/100K corpus comparison with `MODSEC_UNICODE_MAPPING` unset/disabled on the reference side before drawing conclusions about libinjection coverage gaps in Coraza.
